.github/workflows/buildcheck.yml

```diff
@@ -12,12 +12,21 @@ jobs:
         uses: actions/setup-python@v1
         with:
           python-version: 3.8
-
+      - name: Install pyenv
+        run: |
+          # Needed for using exact python version, which is
+          # required in `Pipfile`.
+          curl https://pyenv.run | bash
+          export PATH="$HOME/.pyenv/bin:$PATH"
+          pyenv install 3.8.5
       - name: Install dependencies
         run: |
           python -V
-          pip install -r requirements.txt
-          python setup.py -q install
+          pip install pipenv wheel
+          pipenv install
```

Comment:
you may use
```sh
python -m pip --quiet --no-input install --upgrade pip 
python -m pip --quiet --no-input install --upgrade pipenv wheel
```

The important part would be `--upgrade` and also update `pip` itself. I've found some issues using outdated `pip` in other projects

Reply:
thanks for the pointer. will fix it!

Reply:
Fixed in updated commit